### DISCUSSION TITLE
IMSAI 8080 S-132 board - How to switch VIO character colors and interlaced mode? How to enter VT100 setup mode?

### DISCUSSION THREAD

**Discussion by donfroula** [thehighnibble/thehighnibble.github.io#136] [S-132 add-on for IMSAI & Cromemco replicas]
I'll be darned if I can find instructions for the S-132 on entering the VT100 setup screens, as well as changing the VIO CRT character colors and interlaced mode.

There seems to be lots of information on the VT-132,  but little on the S-132 except on switching terminals, the CPA functions, and mounting and ejecting disk images.

I'd thought there would be better documentation by now!

It seems to me I figured this out at one point, but can't re-discover the info.

Best ,

Don

**Comment by donfroula** [comment]
I think I figured this out.

Part of my confusion was that the Web interface TTY does not allow configuration of the VT100 settings as does the actual S-132 hardware.

Although documented only in the VT-132 docs (nowhere in the S-132 docs), the ALT-printscr combination (when typed on the S-132 keyboard) does bring up the VT100 configuration menu, just like a real VT100. F1 displays a help for that config screen, "5"moves to the next config screen.

Undocumented anywhere is that ALT-printscr in CRT(VIO) mode toggles through the various screen colors in both interlaced and non-interlaced modes.

Best,

Don

**Reply by ronaldlw** [reply]
Any idea how to get the (CRT/VIO) screen color to stick through a power cycle?
